src/main/java/frc/robot/constants/PhoenixDriveConstants.java

```diff
@@ -0,0 +1,182 @@
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+import frc.robot.subsystems.drive.PhoenixDrive;
+
+public final class PhoenixDriveConstants {
+    // Both sets of gains need to be tuned to your individual robot.
+
+    // The steer motor uses any SwerveModule.SteerRequestType control request with the
+    // output type specified by SwerveModuleConstants.SteerMotorClosedLoopOutput
+    public static final Slot0Configs steerGains =
+            new Slot0Configs()
+                    .withKP(150)
+                    .withKI(50)
+                    .withKD(0.2)
+                    .withKS(0.25)
+                    .withKV(1.5)
+                    .withKA(0);
+    // When using closed-loop control, the drive motor uses the control
+    // output type specified by SwerveModuleConstants.DriveMotorClosedLoopOutput
+    public static final Slot0Configs driveGains =
+            new Slot0Configs()
+                    .withKP(0)
+                    .withKI(0.02)
+                    .withKD(0)
+                    .withKS(0.26)
+                    .withKV(0.12)
+                    .withKA(0.01);
+
+    // The closed-loop output type to use for the steer motors;
+    // This affects the PID/FF gains for the steer motors
+    private static final ClosedLoopOutputType steerClosedLoopOutput = ClosedLoopOutputType.Voltage;
+    // The closed-loop output type to use for the drive motors;
+    // This affects the PID/FF gains for the drive motors
+    private static final ClosedLoopOutputType driveClosedLoopOutput = ClosedLoopOutputType.Voltage;
+
+    private static final double kSlipCurrentA = 80;
+
+    public static final double kSpeedAt12VoltsMps = 5.02; // 5.21 OR 5.02
+
+    public static final double autoAlignmentkP = 5.0;
+    public static final double autoAlignmentkI = 5.5;
+    public static final double autoAlignmentkD = 0.0;
+
+    // Every 1 rotation of the azimuth results in kCoupleRatio drive motor turns;
+    // This may need to be tuned to your individual robot
+    private static final double kCoupleRatio = 3.5714285714285716;
+
+    private static final double kDriveGearRatio = 6.122448979591837;
+    private static final double kSteerGearRatio = 21.428571428571427;
+    private static final double kWheelRadiusInches = 1.965;
+
+    private static final boolean kSteerMotorReversed = true;
+    private static final boolean kInvertLeftSide = false;
+    private static final boolean kInvertRightSide = true;
+
+    private static final String kCANbusName = "Canivore";
+    private static final int kPigeonId = 1;
+
+    // These are only used for simulation
+    private static final double kSteerInertia = 0.00001;
+    private static final double kDriveInertia = 0.001;
+    // Simulated voltage necessary to overcome friction
+    private static final double kSteerFrictionVoltage = 0.25;
+    private static final double kDriveFrictionVoltage = 0.25;
```

Comment:
If you want to make the robot less jittery in sim, you could try increasing these values. In any case it would be interesting to see what they do so we can better match the real bot to the sim

Reply:
ill work on that